### ISSUE TITLE
Not working correctly in Penumbra

### ISSUE THREAD

**Issue by Nyococo** [0ceal0t/Dalamud-VFXEditor#96]
Hello! I was working on a mod in VFXEditor and when trying to import it into Penumbra, I notice that for some bizarre reason it is completely ignoring priorities and collections, as all all other people playing the class that's being modified are affected by half of the changes when it was only being used in a collection for myself, even with other mods as higher priority. Strangely, this issue does not persist if applying the mod to others but DOES have a bizarre result; the parts of the mod that were broken for others while enabled for me start to work, while the half that worked previously stop working.

I'm wondering if maybe there is a part of the process of creating the mod that I failed to do for it to work properly in Penumbra. I've tried splitting up the mod into separate skills to troubleshoot (all of them do it regardless), I've tried importing as a Textools mod and as a Penumbra mod, I've tried importing it a second time within Textools, I've tried fiddling with the mod further to see if I can figure it out but it has not gotten anywhere. I tried disabling all other mods in Penumbra itself as well as adjusting priorities, and that seemed to make no difference.

I get the (uneducated) impression that there is something I'm doing wrong in VFXEditor, as it is only now that I've been playing with .tmbs and .paps that I saw this happening. It does not appear to occur when just playing with VFX alone, but I can be wrong.

But lastly, hello! I'm new to GitHub and kinda made the account specifically for this issue, so forgive me if there is anything I may have done incorrectly.

**Comment by 0ceal0t** [comment]
Fixed with latest update (1.7.2.4)

**Comment by Nyococo** [comment]
> Fixed with latest update (1.7.2.4)

A few days after seeing this I updated the plugin, tried replacing Reaper's Slice with Gunbreaker's Keen Edge for a mod I wanted to put together but it still seems to have the same issue. I replaced both the TMB and PAP for reference. I can also try passing along the workspace files if that may help? But honestly I'm pretty sure you can simulate it just by replacing those, assuming this isn't a result of user error on my part.

Regardless, I do appreciate the effort immensely!

**Comment by 0ceal0t** [comment]
hm...I haven't been able to replicate this issue. Was this after you restarted the game after the update?

![image](https://user-images.githubusercontent.com/18051158/225105724-7ea35f4e-5223-42ff-9b0e-6670899fa3ec.png)

**Comment by Nyococo** [comment]
Yep! I patched it one day, came back another day and tried making it. I do notice that you use a different path than I do though, I use cbbm_ws_s01. Would that be related somehow? I had this issue with other skills as well, so I know it's not related to Keen Edge and Slice both using that name for their path

Alternatively I do wonder if maybe something is interacting weirdly with VFXEditor, like perhaps having Penumbra active when exporting might be causing problems. I tried fiddling with this before but didn't seem to have much luck, but maybe there is something I'm missing?

![ffxiv_dx11_NLd7eIsaqN](https://user-images.githubusercontent.com/125221841/225179737-87724b61-dddc-46c8-93b4-ba0fc1c1686a.png)
